k6/prometheus.yaml

```
@@ -0,0 +1,6 @@
+scrape_configs:
+- job_name: backend
+  scrape_interval: 15s
+  static_configs:
+  - targets:
+    - backend:8001
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)